src/lib/y2storage/actions_presenter.rb

```diff
@@ -110,8 +110,7 @@ def dump(file_base_name = nil)
 
     protected
 
-    attr_reader :actiongraph
-    attr_reader :collapsed_subvolumes
+    attr_reader :actiongraph, :collapsed_subvolumes
```

Comment:
I don't see why the new version is better. Not sure if I like the rule.

Nevertheless, a better fix to calm Rubocop down (I guess) would be to document the attributes. Something like

```ruby
# @return [Actiongraph] set of actions to be presented
attr_reader :actiongraph
# @return [Boolean] whether the actions related to subvolumes are currently collapsed
attr_reader :collapsed_subvolumes
```

Reply:
agreed. This was auto correction. Not sure how to force documentation there. 
It is this cop https://msp-greg.github.io/rubocop/RuboCop/Cop/Style/AccessorGrouping.html
And it is also incompatible with sorbet and documentation, so maybe it makes sense to configure it to `separated` style. Unless someone is against.

Reply:
I would simple disable the cop. I see no need to enforce either "grouped" or "separate". Depending on the case one style of declaration or the other may be better (although I personally prefer separate declarations).